src/app/views/playback.py

```diff
@@ -0,0 +1,134 @@
+from pathlib import Path
+
+from PyQt5.QtWidgets import QWidget, QFileDialog
+from PyQt5.QtCore import pyqtSlot, QTimer
+
+from src.app.controllers.playback_controller import PlaybackController
+from src.app.gui.playback_ui import Ui_Playback
+
+
+class Playback(QWidget, Ui_Playback):
+
+    rootDirectory = str(Path(__file__).resolve().parents[3])
```

Comment:
We should get rootDirectory from the mainwindow, instead of creating a new variable. Root directory is already defined in main window.